uc-cdis/gen3-qa
# Feat/export to workspace test

Added tests for `export to workspace` feature, including tests for GUI (portal and JupyterHub) and tests for the manifest service API.

The test plan is specified at here: https://ctds-planx.atlassian.net/wiki/spaces/PLA/pages/42696711/Workspace+Automatic+Integration+Test+Coverage+Plan

Test scenarios including:
- Portal and JupyterHub/Hatchery:
1. Click Workspace tab when logged out and logged in
2. Check export to workspace message toaster
3. Export default manifest, mount it and check manifest name

- Manifest Service API:
1. Export default manifest and check if it exists in manifestservice endpoint

Verified in my dev env that all tests are passing.
